packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -1035,7 +1021,7 @@ export default class CashierStore extends BaseStore {
         this.config.account_transfer = new ConfigAccountTransfer();
         this.config.payment_agent_transfer = new ConfigPaymentAgentTransfer();
         this.is_populating_values = false;
-        this.setIsP2pVisible(false);
+        this.root_store.modules.p2p.setIsVisible(false);
```

Comment:
Ideally in such cases we would want to have the P2P store set this by itself by "reacting" to an "onMount event" of the Cashier store. Is this feasible/worth the overhead here? Same thing with the `onMountCommon` implementation.

Reply:
That makes sense, I've moved all p2p logic to the `p2p-store`.